frontendActions/src/main/resources/actions/messages/GitMacheteBundle.properties

```diff
@@ -0,0 +1,97 @@
+action.checkout.task.title=Checking out
+
+action.fast-forward.description=Fast forward branch ''{0}'' to match ''{1}''
+action.fast-forward.description.root.branch=Root branch ''{0}'' cannot be fast-forwarded
+action.fast-forward.task.title=Fast Forwarding...
+action.fast-forward.text.current-branch=_Fast Forward Parent To Match Current Branch
+
+action.fetch.description=Fetch all remotes
+action.fetch.description.disabled.aleardy-running=Fetch is already running...
+action.fetch.task.title=Fetching...
+
+action.open-git-machete-file.notification.fail.cannot-open=Failed to open machete file
+action.open-git-machete-file.notification.fail.same-name-dir-exists=Cannot create file 'machete': Directory with the same name exists
+
+action.open-git-machete-tab.notification.fail.title=Could not open Git Machete tab
+action.open-git-machete-tab.notification.fail.message.no-git=Ensure that Git VCS is available
+
+action.pull.action-name=P_ull
+action.pull.description-action-name=Pull (fast-forward only)
+action.pull.task.title=Pulling...
+
+action.push.action-name=_Push
+action.push.description-action-name=Push
+
+action.rebase.description=Rebase ''{0}'' onto ''{1}''
+action.rebase.description.disabled.git4idea.repository.status=Cannot rebase ''{0}''
+action.rebase.description.disabled.git4idea.unknown-state=Cannot rebase due to unknown repository state
+action.rebase.description.disabled.merged=Cannot rebase merged branch ''{0}''
+action.rebase.description.disabled.root.branch=Root branch ''{0}'' cannot be rebased
+action.rebase.notification.fail=Rebase failed
+action.rebase.notification.abort=Rebase aborted
+action.rebase.task.title=Rebasing
+action.rebase.hook.task.title=Running machete-pre-rebase hook
+action.rebase.text=_Rebase Branch onto Parent
+
+# used indirectly within BaseResetBranchToRemoteAction
+action.reset.description.enabled={0} branch ''{1}'' to its remote tracking branch
+action.reset.action-name=Re_set to Remote
+action.reset.description-action-name=Reset to remote
+action.reset.task.title=Resetting...
+action.reset.notification.success=Branch ''{0}'' reset to remote
+action.reset.notification.title=Resetting
+
+action.slide-in.dialog.title=Slide In New Branch
+action.slide-in.description=Slide in new branch below ''{0}''
+action.slide-in.description.disabled.no-parent=Slide in disabled due to undefined parent branch
+action.slide-in.task.title=Sliding in
+action.slide-in.text=Slide _In New Branch Below Current Branch
+action.slide-in.notification.fail=Slide in of new branch <b>{0}</b> failed
+
+action.slide-out.description=Slide out ''{0}''
+action.slide-out.description.root.branch=Root branch ''{0}'' cannot be slid out
+action.slide-out.notification.fail=Slide out of <b>{0}</b> failed
+action.slide-out.notification.success=Branch <b>{0}</b> slid out
+
+action.toggle-listing-commits.description=Toggle listing commits
+action.toggle-listing_commits.description.disabled.no-commits=Toggle listing commits disabled: no commits present
```

Comment:
Stray underscore in the key

Reply:
just checking your vigilance :trollface: